src/test/java/seedu/edrecord/model/ModelManagerTest.java

```diff
@@ -37,14 +37,14 @@ public void setUserPrefs_nullUserPrefs_throwsNullPointerException() {
     @Test
     public void setUserPrefs_validUserPrefs_copiesUserPrefs() {
         UserPrefs userPrefs = new UserPrefs();
-        userPrefs.setEdRecordFilePath(Paths.get("address/book/file/path"));
+        userPrefs.setEdRecordFilePath(Paths.get("edrecord/book/file/path"));
```

Comment:
I'm not sure if the `/book/` in the path should also be deleted. Can you help confirm if it's okay? Line 47 also. Thanks 🙏 🙏

Reply:
The path is purely arbitrary, so deleting the `/book/` is fine